apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/service/config/ConfigService.java

```diff
@@ -40,4 +43,11 @@ public interface ConfigService extends ReleaseMessageListener {
    */
   Release loadConfig(String clientAppId, String clientIp, String clientLabel, String configAppId, String
       configClusterName, String configNamespace, String dataCenter, ApolloNotificationMessages clientMessages);
+
+
+  /**
+   * @param releaseKeys
+   * @return the ReleaseMap
+   */
+  ImmutableMap<String, Release> findReleasesByReleaseKeys(Set<String> releaseKeys);
```

Comment:
It's better to use the interface type in the interface, while the implementation can return an ImmutableMap.
```suggestion
  Map<String, Release> findReleasesByReleaseKeys(Set<String> releaseKeys);
```